cmake/clang-tidy.cmake

```diff
@@ -0,0 +1,13 @@
+if (NOT ${CMAKE_SYSTEM_NAME} STREQUAL "Windows")
+    find_program(CLANG_TIDY_EXECUTABLE NAMES clang-tidy-7 clang-tidy)
```

Comment:
clang-tidy-7 is 5 mayor releases old, we should either search also for clang-tidy-{0 ... 12}, or remove it completely

Reply:
Done.